### PR TITLE
Cherry pick: More flexible VPA certificate configuration #1534

### PR DESCRIPTION
**Pull request by bskiba**

### CODE REVIEW

**Comment by bskiba** [comment]
@schylek

**Comment by schylek** [comment]
@jbartosik

File: vertical-pod-autoscaler/pkg/admission-controller/main.go [hunk 1]
```diff
@@ -34,8 +34,13 @@ import (
 )
 
 var (
-	certsDir = flag.String("certs-dir", "/etc/tls-certs", `Where the TLS cert files are stored.`)
```
Comment (jbartosik) [inline vertical-pod-autoscaler/pkg/admission-controller/main.go]: Just to check - do we normally just remove flags? Or do we mark them as deprecated but keep them around for a while before removing?

Comment (bskiba) [reply vertical-pod-autoscaler/pkg/admission-controller/main.go]: Not normally, but I figured this one is unlikely to be used by anyone, and definitely not used by default.

**Pull request merged**